components/eamxx/src/physics/mam/impl/mam4_amicphys.cpp

```diff
@@ -677,13 +677,13 @@ void mam_amicphys_1subarea_clear(
       const int ntot_soamode = 4;
       int niter_out = 0;
       Real g0_soa_out = 0;
-      gasaerexch::mam_gasaerexch_1subarea(
-          nghq, igas_h2so4, igas_nh3, ntot_soamode, gas_to_aer, iaer_so4,
-          iaer_pom, l_calc_gas_uptake_coeff, l_gas_condense_to_mode,
-          eqn_and_numerics_category, dtsubstep, dtsub_soa_fixed, temp, pmid,
-          aircon, num_gas_ids, qgas_cur, qgas_avg, qgas_netprod_otrproc,
-          qaer_cur, qnum_cur, dgn_awet, alnsg_aer, uptk_rate_factor, uptkaer,
-          uptkrate_h2so4, niter_out, g0_soa_out);
+      // gasaerexch::mam_gasaerexch_1subarea(
```

Comment:
These mods are due to the mismatch in the mam4xx submodule. Once microphysics PR is merged, we will rebase to remove these mods.